aether-kernel/aether/kernel/api/tests/test_views.py

```diff
@@ -1013,9 +1013,8 @@ def test_update_by_filters(self):
         )
         response_data = response.json()
         filtered_list = models.Entity.objects.filter(mapping=mapping.pk)
-
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        self.assertIn('is not a valid choice', response_data)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIn('updated', response_data)
```

Comment:
does this mean that any "status" is valid? Not only the two ones we had?

Reply:
@obdulia-losantos, good point. I thought that since we have an enum set in the model for this field that when we do instance.save() we'd get an error. That doesn't seem to be the case. Do we need to perform this validation in the update section of the serializer?

Reply:
Because you are not using the ModelSerializer you need to declare the status field including the choices.

```python
status = serializers.ChoiceField(choices=models.ENTITY_STATUS_CHOICES)
```

https://www.django-rest-framework.org/api-guide/fields/#choicefield